### ISSUE TITLE
Game does not open in headset with the rockstar launcher etc (Steam version)

### ISSUE THREAD

**Issue by JamesDeLaJames** [LukeRoss00/gta5-real-mod#245]
CPU: i5 4670k
GPU: gtx 970
VR headset model: oculus rift
game version and build: latest steam version not sure how to find it
release of the mod: v6
did you follow all the Quick setup steps carefully, in particular step 4? yes
which settings did you choose (H/M/L) when running the batch file RealConfig.bat? M
did the batch file report any errors? no
did you change the graphics settings manually? no
are you using any other mods? no 

Essentially the game opens on desktop but does not open in the rift. If i try to open in steamvr mode it just says the game is not compatible with vr. Yes the theatre mode is unticked.

If i attempt to open by going straight to the exe it wants to open on the headset but then the error saying please open with the rockstar  launcher pops up and I have  no luck opening the game with that and the headset. 

I also later tried a mod to bypass the launcher and it worked but can't get it to work again as the old social club is broken. 

So basically unmodified it won't send to the headset through either oculus or steam vr.

What can I do?

**Comment by JamesDeLaJames** [comment]
I have managed to work around the issue by manually setting VRAPI = 2 such that it opens steam vr. But it only plays in steam vr mode not native oculus.

For me this has fixed my issue unless there is anything else to make  it work on  autodiscover.

**Comment by LukeRoss00** [comment]
Using SteamVR with the Rift will lead to worse performance, so I don't recommend it except as a last resort. `VRAPI = 0` should launch the game on the Oculus runtime without problems: at least it works on my Rift.

Try with `VRAPI = 0`, launching the game from the Steam library (not from the .exe directly and not from the Rockstar launcher) making sure that SteamVR is NOT active.

**Comment by JamesDeLaJames** [comment]
Got it working in the oculus with vrapi= 0, although i'm having pretty random results. I'm also continuously being logged out of the social club and can't run the game now, but i'm assuming that isn't to do with the mod and may need a re install of the game or something.

**Comment by LukeRoss00** [comment]
No problem!

**Comment by Sparadok71** [comment]
I am having a similar issue the game launches but not in VR. I have social club version, the game doesn't show in Oculus app anywhere, the sound comes through Oculus headset but can only see in cinema view. Saw the vrapi = 0 comment but I am not sure what this setting is, sorry new to mods and pc gaming.
